scripts/collect_interest.js

```diff
@@ -1,21 +1,25 @@
 const fse = require("fs-extra");
-const settings = require("./deploy-settings.json");
+const settings = require("../stakingModel/migrations/deploy-settings.json");
 const StakingContract = artifacts.require("./SimpleDAIStaking.sol");
 const GoodFundsManager = artifacts.require("./GoodFundManager.sol");
 
-module.exports = async function(deployer, network) {
-  //currently this is disabled
-  if (network) {
-    console.log("this migration is disabled");
-    return;
+const { networks } = require("../stakingModel/truffle-config.js");
+
+async function getNetworkName() {
+  const networkId = await web3.eth.net.getId();
+  for (let name in networks) {
+    if (networks[name].network_id === networkId.toString())
```

Comment:
this wont work as there are multiple networks that use for example ropsten. (id 3)
you can read network from env like I did (see the script i've uploaded)
or a more correct solution you can read it from process.argv or args ie the arg right after --network

Reply:
d866cf70b148f4de371ffd5ab20be890079ebb88